docs/translating.md

```diff
@@ -17,7 +17,7 @@
 2. After registering check if you got an email to verify your account and click the link (if there is none head to step 1.4)
 3. Log into weblate
 4. Head to https://translate.riot.im/accounts/profile/ and select the languages you know and maybe another language you know too.
-6. Head to https://translate.riot.im/accounts/profile/#subscriptions and select Riot Web as Project
+6. Head to https://translate.riot.im/accounts/profile/#subscriptions and select Element as Project
```

Comment:
Has the name on the translation page been changed to Riot yet?

Furthermore, is there a translation endpoint on element.io?

Reply:
Well, it had not been, but I did it now while reading this PR. 😉

Reply:
Ah and no, there is not a translation service on `element.io` yet.